website/templates/util/render_node.mako

```diff
@@ -119,8 +119,8 @@
 % else:
     <li
         node_reference="${summary['id']}:${'node' if summary['primary'] else 'pointer'}"
-        class="project list-group-item list-group-item-node unavailable">
```

Comment:
Ah, just finished reading this before I merged. Yes, if `unavailable` is unused, let's remove it.